[2025-05-11T12:12:28.531+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-11T12:12:28.582+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_price_btc_price_by_day.save_to_db scheduled__2025-05-09T21:00:00+00:00 [queued]>
[2025-05-11T12:12:28.598+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_price_btc_price_by_day.save_to_db scheduled__2025-05-09T21:00:00+00:00 [queued]>
[2025-05-11T12:12:28.598+0000] {taskinstance.py:2303} INFO - Starting attempt 6 of 6
[2025-05-11T12:12:28.627+0000] {taskinstance.py:2327} INFO - Executing <Task(_PythonDecoratedOperator): save_to_db> on 2025-05-09 21:00:00+00:00
[2025-05-11T12:12:28.635+0000] {warnings.py:110} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=135) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-11T12:12:28.638+0000] {standard_task_runner.py:63} INFO - Started process 136 to run task
[2025-05-11T12:12:28.639+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'get_price_btc_price_by_day', 'save_to_db', 'scheduled__2025-05-09T21:00:00+00:00', '--job-id', '132', '--raw', '--subdir', 'DAGS_FOLDER/get_price_btc_price_by_day.py', '--cfg-path', '/tmp/tmpx1iypznk']
[2025-05-11T12:12:28.643+0000] {standard_task_runner.py:91} INFO - Job 132: Subtask save_to_db
[2025-05-11T12:12:28.751+0000] {task_command.py:426} INFO - Running <TaskInstance: get_price_btc_price_by_day.save_to_db scheduled__2025-05-09T21:00:00+00:00 [running]> on host 840719eb5226
[2025-05-11T12:12:28.999+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='get_price_btc_price_by_day' AIRFLOW_CTX_TASK_ID='save_to_db' AIRFLOW_CTX_EXECUTION_DATE='2025-05-09T21:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='6' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-09T21:00:00+00:00'
[2025-05-11T12:12:29.001+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-11T12:12:29.024+0000] {base.py:84} INFO - Using connection ID 'etl_db' for task execution.
[2025-05-11T12:12:29.079+0000] {get_price_btc_price_by_day.py:68} INFO - Данные записались успешно!
[2025-05-11T12:12:29.080+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-11T12:12:29.080+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-11T12:12:29.105+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=get_price_btc_price_by_day, task_id=save_to_db, execution_date=20250509T210000, start_date=20250511T121228, end_date=20250511T121229
[2025-05-11T12:12:29.139+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-11T12:12:29.193+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-11T12:12:29.196+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
